co2calculator/calculate.py

```diff
@@ -587,13 +588,17 @@ def calc_co2_businesstrip(
         stops = None
     elif start is not None and destination is not None and distance is None:
         # check if stops are provided in the right form
+        # NOTE: transportation_mode 'car' requires string type for start and destination
         if transportation_mode == "car" and (
             type(start) != str or type(destination) != str
         ):
-            raise ValueError(
-                "Wrong data type for start and destination."
-                "Please provide a three letter IATA code for train stations."
-            )
+            # NOTE: I turned off that type check since lower level functions expect dicts
+            # (calc_co2_car & geo_coding_structured both want dictionaries)
+            pass
```

Comment:
@han16nah Last git changes are referencing you :)

I had to turn off this check to make my functional test work. I was wondering though if I missed something.

My analysis showed:

In the event of `calc_co2_businesstrip()` with `transportation_mode = "car"`, it checks for type str for `start` and `destination`.

BUT: Both lower-level functions - `calc_co2_car` and `geocode_structured` - require a dictionary.

It actually works with disabling the checks (see `tests/functional/test_calculate.py`.


My question is: is `calculate_co2_businesstrip` used atm and if .. does is work? 😂

Reply:
Huh, it seems like something is very messed up here.
I guess "car" would need to be replaced with "plane" because that's where a string with the IATA code is needed.. 
https://github.com/pledge4future/co2calculator/blob/67592506a59431e3cbee1645d8a52196efd4e0b4/co2calculator/calculate.py#L333
And I don't know what's up with the Error message then :D this should obviously be plane as well

Should I fix something and push to dev or will you just make the changes in your branch?

PS: Nah, probably does not work ;)

Reply:
How about this: I finish functional tests first. If necessary, comment and fix things. If that's all done, I need you (or someone else) to review if the test data is what the backend really sends. 

If the tests are set up, I gonna apply changes to distance calculation and input validation anyway!